src/components/layout/Layout.tsx

```diff
@@ -26,6 +27,8 @@ export const Layout: FC<ContainerProps> = ({ children, ...props }) => {
         children
       )}
 
+      <Forms />
+
       <Box px={5} pb={3}>
         <UnicornSpace />
```

Comment:
Probably I'm outdated with this but in the designs I see a different image here:
![image](https://user-images.githubusercontent.com/468158/149629676-b703917a-8ee4-4542-82cb-74b87db7e07b.png)

Not something in the scope of this PR but I wanted to make sure to point it out.

Reply:
Yes, haven't change that image because it wasn't defined yet (not sure if that's already the final version, is it @tomosaito?) , I'm keeping that change for later.